snippets/arduino.cson

```diff
@@ -97,6 +100,21 @@
   'Shift Out':
     'prefix': 'shiftOut'
     'body': 'shiftout(${1:dataPin}, ${2:clockPin}, ${3:bitOrder}, ${4:value});$5'
+  'Switch':
+    'prefix': 'switch'
+    'body': """
+      switch (${1:variable}) {
```

Comment:
I'm not a fan of adding non-arduino specific snippets. Switch is a C/C++ statement.

Reply:
Currently snippets aren't imported from the C/C++ language package and C/C++ snippets can't be used with this Arduino language because the C package doesn't know this exists. Unless Atom's C/C++ language adds `.source.arduino` to the header of their snippet file, users won't be able to use those snippets with this language package.

Reply:
Or we could treat this package as an addition the the C++ language package and tell its snippets and syntax to bind to anything that uses the C++ grammar. I'll make a separate branch to test that and remove the switch statement in this PR.

Reply:
@szechyjs Done. Check out #22